src/models/project.js

```diff
@@ -0,0 +1,13 @@
+export default class Project {
+  constructor(name) {
+    this.name = name;
+    this.todos = [];
+    let projectCount = Number(localStorage.projectCount);
+    this.id = ++projectCount;
+    localStorage.setItem("projectCount", projectCount);
```

Comment:
The model `Project` should not be aware about the underlying storage, nor how projects are incremented! It should strictly create a new project model, and return it. 

Incrementing the projects count is probably a responsibility of the Projects Controller.